.github/workflows/wheels.yml

```diff
@@ -0,0 +1,76 @@
+# Workflow to build and test wheels
+name: Wheel builder
+
+on:
+  schedule:
+    # Nightly build at 3:42 A.M.
+    - cron: "42 3 */1 * *"
+  push:
+    tags:
+      # Stable release build
+      - "[0-9]+.[0-9]+.[0-9]+"
+  pull_request:
+    branches:
+      - master
+      - "[0-9]+.[0-9]+.X"
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Checkout scikit-learn
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+    
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==1.5.2
+
+      - name: Build wheels
+        env:
+          # Build the wheels for Python 3.6 and newer
+          CIBW_SKIP: pp* *p27* *p35*
+          CIBW_TEST_REQUIRES: pytest
```

Comment:
I think you need to add the test command for tests to actually run,
```
CIBW_TEST_COMMAND: "pytest --pyargs sklearn"
```
we could probably also add pandas to `CIBW_TEST_REQUIRES`.

Reply:
You are absolutely right @rth, I forgot the command to run the tests.